includes/fields/class-field.php

```diff
@@ -220,11 +220,6 @@ public function __construct( $args = [] ) {
 	 */
 	protected function boot() {
 
-		// Set default value.
```

Comment:
Please avoid changing base API like this, removing this may affect not only Attributes but any place where we (or third-party developers) used Field classes, e.g. in HivePress settings (these are not related to the attribute logic), meta boxes, etc. Find a way to implement this without removing this part.